aries-site/src/components/feedback/Question.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  FormField,
+  TextArea,
+  Text,
+  RadioButtonGroup,
+  FileInput,
+} from 'grommet';
+import { StarRating } from './StarRating';
+import { ThumbsRating } from './ThumbsRating';
+
+export const Question = ({ formProps, label, inputProps, kind, name }) => {
+  const accessibilityProps = {
+    id: name,
+    name,
+  };
+
+  const formats = {
+    star: { render: StarRating, border: false },
+    thumbs: {
+      render: ThumbsRating,
+      border: false,
+      label: <Text size="large">{label}</Text>,
+    },
+    textArea: { render: TextArea },
+    radioButtonGroup: { render: RadioButtonGroup },
+    fileInput: { render: FileInput },
+  };
+
+  const Input = formats[kind].render;
+  return (
+    <FormField
+      contentProps={{
+        border: formats[kind].border ?? true,
+      }}
+      label={formats[kind].label || <Text>{label}</Text>}
```

Comment:
Why are we not using default formfield styling for label?

Reply:
Ill change your right

Reply:
https://github.com/grommet/hpe-design-system/pull/2839#discussion_r970286914

Reply:
@taysea @britt6612  my recommendation ☝️  is to make all question text uniform. I believe the feedback context is unique from a form context and might deserve a modified treatment.

![Screen Shot 2022-10-03 at 3 13 31 PM](https://user-images.githubusercontent.com/1756948/193691214-1c018f27-7d38-43bd-a083-867ecb1183f1.png)